libraries/mongodb.rb

```diff
@@ -80,17 +87,11 @@ def self.configure_replicaset(node, name, members)
       "Configuring replicaset with config: #{rs_members}"
     )
 
-    rs_member_ips = []
-    members.each_index do |n|
-      port = members[n]['mongodb']['config']['port']
-      rs_member_ips << { '_id' => n, 'host' => "#{members[n]['ipaddress']}:#{port}" }
-    end
-
     admin = connection['admin']
     cmd = BSON::OrderedHash.new
     cmd['replSetInitiate'] = {
       '_id' => name,
-      'members' => node['mongodb']['use_ip_address'] ? rs_member_ips : rs_members,
+      'members' => rs_members,
```

Comment:
ah, shame on me for skimming to quickly!

and I was thinking of #165, which actually is terrible.

But yay for consistency and handling related things in the same place!